src/chi-vue/src/views/DataTable/ClientSide/DataTableClientView.vue

```diff
@@ -137,8 +184,21 @@ const MOCK_API_RESPONSE_DELAY = 5000;
     chiColumnsReset: e => {
       console.log('chiColumnsReset', e);
     },
-    chiSelectedRowsChange: e => {
-      console.log('chiRowSelect', e);
+    chiSelectedRowsChange(data) {
+      this.$data.bulkActionConfig = {
+        ...this.$data.bulkActionConfig,
+        showBulkActions: true,
```

Comment:
Since the implementation makes Bulk action an optional module of Data Table, is this `showBulkActions` necessary here?

Reply:
Not necessary. Updated as required.